### ISSUE TITLE
Specify custom cURL options

### ISSUE THREAD

**Issue by jkrehm** [twilio/twilio-php#362]
In 4.x, the `Services_Twilio_TinyHttp` class allowed you to set your own cURL options. I don't see a way to do the same with the `Twilio\Http\CurlClient`. Am I overlooking it? Was it intentionally removed/not added? If it wasn't intentional, would you be willing to accept a PR to provide the capability?

**Comment by ihumanable** [comment]
This was not intentionally removed, probably an oversight, I will look into putting this back in.  Thank you for your report.

**Comment by jkrehm** [comment]
Thanks so much!

**Comment by ihumanable** [comment]
Here is a PR that implements this behavior https://github.com/twilio/twilio-php/pull/363

Once that lands you should be able to create a custom `Twilio\Http\CurlClient` and pass it into the constructor of `Twilio\Rest\Client` like so

```php
<?php

using Twilio\Http\CurlClient;
using Twilio\Rest\Client;

$curlClient = new CurlClient(array(CURLOPT_VERBOSE => true));  // Replace with whichever options you wish 
$client = new Client('AC123', 'abc123', null, null, $curlClient);
```

**Comment by ihumanable** [comment]
@jkrehm This has landed and is released as twilio/sdk 5.0.3
